controllers/comments-controller.js

```diff
@@ -0,0 +1,17 @@
+//REQUIRES
+const { selectComments } = require("../models/comments-model");
+
+//FUNCTION
+function getComments(req, res, next) {
+  selectComments(req.params.article_id)
+      .then((comments) => {
+      res.status(200).send(comments);
```

Comment:
Please make sure you are retuning the data on an appropriate key e.g. {comments: comments}